source/mainnet/smart-contracts/tutorials/piggy-bank/testing.rst

```diff
@@ -237,6 +237,16 @@ Then compile your contract with:
 This produces the Wasm module ``piggy_bank_part2.wasm.v1`` in the root of your project, unless you have any typos or bugs in your code.
 If that is the case, try to fix them using the helpful error messages from the compiler or go back to the end of part 1 and copy the full contract code again.
 
+.. note::
+
+   Since the tests run against the compiled Wasm module, there is a risk of accidentally using an outdated Wasm module.
+   To circumvent this, cargo concordium's test command can both *build* and test you contract, which ensures that you always test the newest version of your code.
```

Comment:
Perhaps we could add here that if you use `cargo concordium test --out` then you 
- build the module and output it
- and run tests on it.

And thus you should deploy that module, which means you are deploying a module you have tested, exactly that one.

Reply:
Yeah, I think that's good.